source/blog/2018-11-09-the-ember-times-issue-72.md

```diff
@@ -9,7 +9,7 @@ responsive: true
 <SAYING-HELLO-IN-YOUR-FAVORITE-LANGUAGE> Emberistas! 🐹
 
 <SOME-INTRO-HERE-TO-KEEP-THEM-SUBSCRIBERS-READING>.
-We need **Emberistas at JSConf EU**, so submit a talk to the newly opened CFP! 🗣️
+We need **Emberistas at JSConf EU**, so submit a talk to the newly opened CFP! 🗣️ Let's take a look into **more informative Routes** 🤓,
```

Comment:
Maybe this should be on a new line? Doesn't really flow with the sentence.

Reply:
We workshop the intro at the end, so it seems ok for now! We can edit it as we get more writeups / potentially rearrange the order of articles...